tests/serverspec/default.yml

```diff
@@ -2,4 +2,32 @@
   roles:
     - ansible-role-nginx
   vars:
-    nginx_config: ""
+    nginx_flags: -q
```

Comment:
i am writing another test case for TLS configuration, and it will be tested in the case.

btw, `nginx -q` is irrelevant in this context. `-q` disables non-error messages in stdout.